03week/airplane/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+  <html>
+    <head>
+      <link rel="stylesheet" href="./css/style.css">
+      <title>Box Model Assignment</title>
+    </head>
+    <body>
+      <header>
+        <section class="Logo">
+          <img src="./img/logo.png"/>
+        </section>
+        <section class="navbar">
+          <img src="./img/home-icon.png"/>
+          <span home>Home</span>
+          <img src="./img/deal-icon.png"/>
+          <span travel>Travel Deals</span>
+          <img src="./img/schedule-icon.png"/>
+          <span flight>Flight Schedule</span>
+          <img src="./img/login-icon.png"/>
+          <span login>Register/Login</span>
+        </section>
+      </header>
+      <main>
+        <aside>
+          <section plane-icon>
+            <img src="./img/plane-icon.png"/>
+          </section>
+          <section hotel-car-icon>
+            <img src="./img/hotel-icon.png"/>
+            <img src="./img/car-icon.png"/>
+          </section>
+          <section>
+            <input type="Depart" placeholder="Depart">
```

Comment:
type should really be "text", but you could add a different attribute to hold "Depart". or maybe make it the id.